pom.xml

```
@@ -22,7 +22,7 @@
 
         <!-- dependencies -->
         <quarkus.version>3.2.2.Final</quarkus.version>
-        <karate.version>1.4.0</karate.version>
+        <karate.version>1.4.1</karate.version>
         <graal.version>22.3.0</graal.version>
```

```suggestion
        <graal.version>22.3.3</graal.version>
```
Sync with version in this Karate release